python/deltalake/table.py

```diff
@@ -456,6 +456,27 @@ def vacuum(
             enforce_retention_duration,
         )
 
+    def update(
+        self,
+        updates: Dict[str, str],
+        predicate: Optional[str] = None,
+        writer_properties: Optional[Dict[str, int]] = None,
+        strict_cast: bool = True,
+    ) -> Dict[str, Any]:
+        """UPDATE records in the Delta Table that matches an optional predicate.
```

Comment:
Could we add an examples section ([like this](https://github.com/delta-io/delta-rs/blob/4da7d66d06985d386e61d3fb124cad6680594dcc/python/deltalake/table.py#L649-L665))? I feel like it might not be obvious how to use the `updates` param in particular.

If you are willing, it would be awesome to have a section in https://github.com/delta-io/delta-rs/blob/main/python/docs/source/usage.rst

Reply:
Will do! Forgot to add it

Reply:
Added also section in the docs.